main.tf

```diff
@@ -23,6 +23,8 @@ resource "azurerm_kubernetes_cluster" "main" {
   disk_encryption_set_id              = var.disk_encryption_set_id
   dns_prefix                          = var.prefix
   http_application_routing_enabled    = var.http_application_routing_enabled
+  image_cleaner_enabled               = var.image_cleaner_enabled
+  image_cleaner_interval_hours        = var.image_cleaner_enabled && var.image_cleaner_interval_hours != null ? var.image_cleaner_interval_hours : null
```

Comment:
Could we use `image_cleaner_interval_hours = var.image_cleaner_interval_hours` here?